test/tools/yulInterpreter/Interpreter.h

```diff
@@ -102,6 +102,12 @@ struct InterpreterState
 class Interpreter: public ASTWalker
 {
 public:
+	struct Scope
+	{
+		std::map<YulString, FunctionDefinition const*> functions;
```

Comment:
It is probably possible to remove this member (and thus also replace the struct by the symbol set below) and use `m_functions` for the lookup inside `ExpressionEvaluator::visibleFunctionsFor`.